src/benchcab/fluxsite.py

```diff
@@ -60,6 +61,20 @@ def __init__(
         self.sci_config = sci_config
         self.logger = get_logger()
 
+    def _get_state_dir(self) -> Path:
```

Comment:
This and the one above should be turned into a common utility method that takes the suffix as an argument. Less redundant code. :)